data/news/articles/c0g992jmmkko.json

```diff
@@ -672,7 +672,9 @@
     }
   },
   "metadata": {
-    "analyticsLabels": {},
+    "analyticsLabels": {
+      "nations_producer": "scotland"
+    },
```

Comment:
Just to check, this new producer field is only for Optimo articles right?

Reply:
Yep only public service Optimo articles (produced by one of the 4 nations teams)